### ISSUE TITLE
`make lesson-check` fails on Win10 machine

### ISSUE THREAD

**Issue by hoytpr** [carpentries/styles#447]
In GitBash: On a completely new clone of lesson-example, named `data-cleanup` as in the instructions:
Earlier noticed that `bin\lesson-initialize.py` failed with `python3:permission denied` which is a cryptic message, but solved (#283 ) by running `$ python bin\lesson-initialize.py`.

Python 37 is installed under `Users/username/Anaconda3/python.exe` (which is in the Windows path) and a new environment variable named `python3` was created with the value `Users/username/Anaconda3/python.exe`, however this does not allow `bin\lesson-initialize.py` or any python script to be run without prepending the script with `python`. 

NEW issue: Running **`make lesson-check`** also fails (perhaps related to carpentries/lesson-example#249 ) but this error message was:
```
$ make lesson-check
index.md:6:FIXME: home page introduction
index.md:14:> FIXME
setup.md:4:FIXME
_episodes/01-introduction.md:6:- "Key question (FIXME)"
_episodes/01-introduction.md:8:- "First learning objective. (FIXME)"
_episodes/01-introduction.md:10:- "First key point. Brief Answer to questions. (FIXME)"
_episodes/01-introduction.md:12:FIXME
reference.md:7:FIXME
_extras/discuss.md:4:FIXME
_extras/guide.md:4:FIXME
make: bin/lesson_check.py: Command not found
make: *** [Makefile:96: lesson-check] Error 127
```
This suggests that the make command works until another python script `bin/lesson-check.py` is called. I am not experienced with makefiles, but it appears to call several python scripts. Each of these would need to be prepended with `python` to work on my machine. Any advice is appreciated.

**Comment by brownsarahm** [comment]
Thanks @hoytpr for raising this. I moved it to styles because I think you actually followed the instructions on lesson-example and  cloned this repo, not lesson-example itself.  If that's not the case, please try that and then update us. 

However, I think your assessment of the issue relating to how python is called is correct. A solution for several windows compatibility issues is in the works, but not yet merged into this repo. 
This should be solved by #446. We can let you know when that's merged so you can pull those changes and try again.

**Comment by hoytpr** [comment]
Thank you @brownsarahm for your quick attention to this. Yes, I used GitHub’s importer to import this repo in my GitHub account using https://github.com/carpentries/styles. I then cloned this locally exactly as described in the setup instructions. I was trying to reproduce #445 on my machine as it's a rare opportunity to get another Windows user to compare notes with. The entire process (after importing from the styles repo) is below:
```
$ git clone -b gh-pages https://github.com/hoytpr/data-cleanup.git
$ cd data-cleanup/
$ git remote add template https://github.com/carpentries/styles.git
$ git config --local remote.template.tagOpt --no-tags
$ git checkout gh-pages
$ bin/lesson_initialize.py
($ python bin/lesson_initialize.py)  # had to prepend "python"
$ ls
_config.yml    _extras    aio.md   bin       CODE_OF_CONDUCT.md  fig       LICENSE.md  reference.md
_episodes      _includes  assets   CITATION  CONTRIBUTING.md     files     Makefile    setup.md
_episodes_rmd  _layouts   AUTHORS  code      data                index.md  README.md
$ make lesson-check
index.md:6:FIXME: home page introduction
index.md:14:> FIXME
setup.md:4:FIXME
_episodes/01-introduction.md:6:- "Key question (FIXME)"
_episodes/01-introduction.md:8:- "First learning objective. (FIXME)"
_episodes/01-introduction.md:10:- "First key point. Brief Answer to questions. (FIXME)"
_episodes/01-introduction.md:12:FIXME
reference.md:7:FIXME
_extras/discuss.md:4:FIXME
_extras/guide.md:4:FIXME
make: bin/lesson_check.py: Command not found
make: *** [Makefile:96: lesson-check] Error 127
```

**Comment by maxim-belkin** [comment]
> make: bin/lesson_check.py: Command not found
> make: *** [Makefile:96: lesson-check] Error 127

This issue should be solved when we merge #446.
